zh_CN/用户手册/DSS新增用户方式.md

```diff
@@ -4,34 +4,50 @@
 
 ## 一、基本介绍
 
- DSS 超级管理员的用户名为部署用户名，如部署用户为 hadoop，则管理员的用户名和密码为 hadoop/hadoop，具体用户可在 [DSS单机部署文档](../安装部署/DSS&Linkis一键部署文档单机版.md) 查看。
+DSS 超级管理员的用户名为部署用户名，如部署用户为 hadoop，则管理员的用户名和密码为 hadoop/hadoop，具体用户可在 [DSS单机部署文档](../安装部署/DSS&Linkis一键部署文档单机版.md) 查看。
 
 新增一个 DSS 用户主要分为以下几步：
 
+- DSS接入LDAP认证体系
 - 新增 LDAP 用户
 - 为新用户完善环境信息
 
-## 二、新增 LDAP 用户
-
- DSS 超级管理员可以在首页创建部门和用户，建立公司的层级及人员管理体系，创立的部门和用户。如下图所示：
- 
- ![超级管理员首页](images/超级管理员功能.png)
- 
- #### 创建部门和用户：
- 
- 在超级管理员首页点击【管理台】，进入管理台页面。
- 
- 超级管理员可以新建、修改和删除部门（请注意：部门层级最多可建立四级，包括元公司在内），元公司无法删除，仅可修改信息。
- 
- 同时，超级管理员可以进行用户的新建，修改和用户密码重置。
- 
- 如下图所示：
- 
-  ![创建部门和用户](images/创建部门和用户.png)
-
- 当您在页面创建一个用户后，DSS 后台会自动请求 LDAP，为您在 LDAP 中创建一个同名用户。
-
-## 三、为新用户完善环境信息
+## 二、DSS接入LDAP认证体系
+
+1.首先要搭建好LDAP服务端，确保LDAP服务可用，能正常新增用户。如下图所示（以ldapadmin客户端展示为例）：
```

Comment:
It is better if we can add a message about how to install LDAP, link the LDAP installation page.

Reply:
ok ,   i will supplement a page to install LDAP